### PR TITLE
Update Developers Docs

### PR DESCRIPTION
**Pull request by arthurpar06**
I changed many things, so it would be great if someone could review it. I was careful, but I probably left some English mistakes as well.

### CODE REVIEW

File: docs/developers/environment.md [hunk 1]
```diff
@@ -13,86 +13,85 @@ This is just how I work on phpVMS, to quickly reset the database and test/develo
...
+:::info
+You may want to disable caching during development phases to ensure your changes are applied correctly, as caching cna prevent changes from taking effect. 
+To do this, set `CACHE_DRIVER=null` in your `.env` file.
+
+:::
 
-:::caution
-This requires sqlite to be installed on your system
+:::info
+We strongly recommend reading the [Laravel Documentation](https://laravel.com/docs/11.x), as it's more detailed and up-to-date. Since phpVMS is based on Laravel, the installation process is the same.
```
Comment (FatihKoz) [inline docs/developers/environment.md]: We are still using Laravel 10, I think this should point the correct version until we switch.

**Comment by FatihKoz** [comment]
If all these are designed for Laravel 11 then we should wait for it, if not we should provide other alternatives for development environments and provide links for what we have at the moment.

One may not wish to have docker, does not want to use Laravel Sail etc. In theory, these documents are here to help people while creating modules and/or supporting their VA's.

Of course it will not be possible to cover and document all possible environment alternatives but there should be at least some references here that there are alternatives and what is written here is not a "must" for developing stuff for phpvms v7.

Like you can simply have xammp, wammp, laragon etc on your Windows and easily develop stuff without following any of these steps/topics in this guide.

* A running local http server, mysql/maria server
* Matching php version, composer and maybe node for packages
* Access to matching Laravel Docs

Just my general thoughts.

**Comment by arthurpar06** [comment]
Good catch with the links, I didn't notice that but i'll change them in the afternoon :)

Also, I completely agree that Docker isn't required. There is indeed a "Without Docker" section in what I'm proposing, but it might be a bit short, we could maybe add some more information.

This is what's currently in the PR. Let me know what you'd like to add.


EDIT: After rereading, it also needs to be clarified that it's really the dev-install command that requires sqlite, not composer or anything else.

image

**Comment by FatihKoz** [comment]
Yeah I mean it starts with a full config and wording about it :) The way how Nabeel works, it is nice for people with knowledge but I think this page should be more generic.

It should start with a basic explanation about it is possible to have a local environment for tests and development, and then it should go on with basics and details like

Introduction
Basic Requirements
Simple Setup (without docker can be placed maybe here)
Expanded Setup (with docker, more dev stuff etc)

If a normal guy, non-dev but with some needs, want to learn, he/she should not be blown up with tons of tech stuff in the first lines of a doc. An experienced dev can skip them and move forwards but normal people (with some basic knowledge) not :)

**Comment by FatihKoz** [comment]
By the above I mean the complete section should be revised, not only this page 😉 

Building assets, addons, awards all of them. Then we can have a nice documentation for development & enhancements.

**Comment by arthurpar06** [comment]
Yeah, I agree. Writing clear documentation is really not easy... at least I find it difficult. But your idea is good - creating something clean that explains both the basics for those who need them and the technical details for those who want to do more advanced stuff is a nice idea.

**Pull request merged**